app/templates/community/invasion-of-ukraine.hbs

```diff
@@ -0,0 +1,129 @@
+{{page-title "Invasion Of Ukraine"}}
+
+<section aria-labelledby="hero-title" class="bg-shape-boxes-bottom bg-dark temp-fullpage">
+  <div class="container layout">
+    <div class="lg:col-5">
+      <h1 class="text-hero-xl" id="hero-title">
+        What is happening in Ukraine?
+      </h1>
+      <h3>
+        DEAR FRIENDS
+      </h3>
+      <p class="text-hero-base mb-2">
+        These days the world's media focus upon Ukraine.
+        We understand that not all people may know what is happening in Ukraine.
+        We have collected information to help you better understand what the Ukrainian people are
+        now experiencing.
+      </p>
+      <ul>
+        <li class="text-hero-base mb-2">
+          On February 24, around 5 a.m., Russia began the full-scale military invasion of Ukraine.
```

Comment:
```suggestion
        <li class="text-hero-base mb-2">
          On February 24th, around 5 a.m., the Russian military began the full-scale military invasion of Ukraine based on orders from Vladimir Putin, the president of Russia.
```

Let's add some clarity that it was the Russian military at the direction of the Russian president.